host/4/mariner/dotnet/dotnet-isolated/dotnet7-isolated.Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Build the runtime from source
+ARG HOST_VERSION=4.21.1
+FROM mcr.microsoft.com/dotnet/sdk:6.0-cbl-mariner2.0 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest=false
+
+RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
+    git clone --branch v${HOST_VERSION} https://github.com/Azure/azure-functions-host /src/azure-functions-host && \
+    cd /src/azure-functions-host && \
+    HOST_COMMIT=$(git rev-list -1 HEAD) && \
+    dotnet publish -v q /p:BuildNumber=$BUILD_NUMBER /p:CommitHash=$HOST_COMMIT src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj -c Release --output /azure-functions-host --runtime linux-x64 && \
+    mv /azure-functions-host/workers /workers && mkdir /azure-functions-host/workers && \
+    rm -rf /root/.local /root/.nuget /src
+
+# Include ASP.NET Core shared framework from dotnet/aspnet image.
+FROM mcr.microsoft.com/dotnet/aspnet:7.0-cbl-mariner2.0 AS aspnet7
+
+FROM mcr.microsoft.com/dotnet/runtime:7.0-cbl-mariner2.0
+ARG HOST_VERSION
+
+RUN yum install -y dnf && \
```

Comment:
I think that the installs here are likely not necessary. Could be removed but not necessary

Reply:
Changed